packages/portfolio-contract/src/portfolio.contract.ts

```diff
@@ -62,6 +80,8 @@ export const contract = async (
   const { makeLocalAccount, openPortfolio } = orchestrateAll(flows, {
     zoeTools,
     makePortfolioKit,
+    chainHub,
+    contract: privateArgs.contract,
```

Comment:
hm... the name `contract` doesn't look so good here, does it?

Reply:
I still need to know where the contract addresses come from. Are they the same in all configurations? i.e. in testnets as well as production?